### ISSUE TITLE
murmurhash3 seems to compile native version to wrong location

### ISSUE THREAD

**Issue by tmaier** [funny-falcon/murmurhash3-ruby#10]
I just installed murmurhash3 with `gem install murmurhash3` on my computer (tested on mac and linux., intel-based)

I receive the following message:

```ruby
require 'murmurhash3'
# Attention: used pure ruby version of MurmurHash3
# => true
```

I checked the gem location on my computer. The `native.so` has been created at  `./lib/native.so`

![image](https://user-images.githubusercontent.com/350038/210184149-52d3d80a-7d4c-498a-b401-e0d4af4f9b06.png)

The reason for this error is, that it cannot require the file `murmurhash3/native`:

```ruby
require 'murmurhash3/native'
# LoadError: cannot load such file -- murmurhash3/native
# from /Users/myuser/.rbenv/versions/2.7.6/lib/ruby/site_ruby/2.7.0/rubygems/core_ext/kernel_require.rb:85:in `require'
```

I was able to fix this error and also the warning above by moving the `native.so` file to `./lib/murmurhash3/native.so`

```ruby
require 'murmurhash3'
# => true
```

**Comment by tmaier** [comment]
I just checked the forks available. It seems like @jirutka identified this issue as well and fixed it at https://github.com/jirutka/murmurhash3-ruby/commit/7208d367b22f6518f520664b57955392e7a7629a (I did not test it myself)

**Comment by wildmaples** [comment]
We're encountering the same problem and noticed it only when we rebuilt the gem. I'm not sure why it's happening now since this gem hasn't been updated in a while. It could be related to how Bundle compiles the native extension.

**Comment by funny-falcon** [comment]
I've merged #9 . Now I'll try to remember how to publish.

**Comment by funny-falcon** [comment]
Pushed 0.1.7
